tests/interoperability_test.py

```diff
@@ -389,5 +415,36 @@ def test_with_seq_corr_one_set_generator_non_noised_different_sizes(self):
     self.simulate_with_set_generator(set_generator_factory,
                                      self.name_to_noised_estimator_config)
 
+  def test_run_evaluation_works(self):
```

Comment:
This unit test name seems a little opaque.  According to Google's guidelines, "Test names should include the condition being tested and the expected outcome."  Could possibly be addressed by adding a comment explaining further, but probably the best practice would be to make the name more descriptive.

Reply:
I changed the test name and add some docstring to explain more details. PTAL. Thanks!